autoscheduler/scraper/models/course.py

```diff
@@ -1,17 +1,27 @@
 from django.db import models
 
-class Course(models.Model): 
+def generate_course_id(dept: str, course_num: str, term: str):
+    """ Generates the coures_id in the form of {dept}{course_num}-{term} """
+
+    return "".join((dept, course_num, '-', term))

```

Comment:
Super simple function, thought join was the most more readable than doing `f'{dept}{course_num}-{term]'`, could also change to `dept + course_num + '-' + term`, bit of a nitpick though